lib/pliny/log.rb

```diff
@@ -72,6 +72,14 @@ def stderr
       @stderr
     end
 
+    def scrubber=(s)
```

Comment:
Ah, right. I thought about it in terms of `Pliny::Log` but, yes, because `Pliny` gets extended with `Pliny::Log`, yes? Good point.